pkgs/by-name/li/libparse-python/package.nix

```
@@ -0,0 +1,57 @@
+{
+  pkgs,
+  callPackage,
+  fetchFromGitHub,
+}:
+
+let
+  pname = "libparse-python";
+  version = "unstable-2025-12-15";
+in
+
+pkgs.python3Packages.buildPythonPackage {
+  inherit pname version;
+
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "librelane";
+    repo = "libparse-python";
+    rev = "ff5b26d";
+    hash = "sha256-kNCLxr5FKqXzpbKF4ogoFdthLI4S/3r9QxT5DXSVziw=";
+  };
```

Use the full commit hash for better reproducibility and fetch submodules, so we don't have to patch `yosys` anymore.


```suggestion
  src = fetchFromGitHub {
    owner = "librelane";
    repo = "libparse-python";
    rev = "ff5b26da3d66affb60696fd6c4af7f8726729c8c";
    hash = "sha256-H2UHh5RbxJ65yTbLybCEanl5qA4o0e/0rAQKmIhvXeQ=";
    fetchSubmodules = true;
  };
```